src/Elastic.Apm.AspNetFullFramework/ElasticApmModule.cs

```diff
@@ -0,0 +1,226 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Runtime.InteropServices;
+using System.Web;
+using Elastic.Apm.Api;
+using Elastic.Apm.DiagnosticSource;
+using Elastic.Apm.DistributedTracing;
+using Elastic.Apm.Logging;
+using Elastic.Apm.Model;
+
+namespace Elastic.Apm.AspNetFullFramework
+{
+	public class ElasticApmModule : IHttpModule
+	{
+		private static readonly bool _isCaptureHeadersEnabled;
```

Comment:
nit: should be `IsCaptureHeadersEnabled` according to `.editorconfig`

Reply:
Fixed.